source/ranges.tex

```diff
@@ -892,8 +892,8 @@
 
 \pnum
 The \libconcept{sized_range} concept specifies the requirements
-of a \libconcept{range} type that knows its size in constant time with the
-\tcode{size} function.
+of a \libconcept{range} type that knows its size in constant time with
+\tcode{ranges::size}.
```

Comment:
Can we maybe fix this a bit more? "knows its size" doesn't sound like a specification, and "with" is a bit non-specific. Maybe "of a range type whose size can be determined in constant time using ranges::size." or so.

Reply:
Sure.